metadata/fast-switcher.xml

```diff
@@ -7,7 +7,7 @@
 		<option name="activate" type="key">
 			<_short>Activate</_short>
 			<_long>Activates fast switcher with the specified key.</_long>
-			<default>&lt;alt&gt; KEY_TAB</default>
+			<default>&lt;alt&gt; KEY_ESC</default>
```

Comment:
Alt+Tab is a very common binding for switching windows and should be included.

Reply:
I think alt-tab should go to switcher actually, and here alt-esc as proposed.